model.py

```diff
@@ -3,6 +3,9 @@
 from flask_sqlalchemy import SQLAlchemy  # connection to PostgreSql database
 # getting this from the Flask-SQLAlchemy helper library
 
+# Example project for full usage demo: https://github.com/code-workshops/blogwise
+from werkzeug.security import generate_password_hash, check_password_hash
```

Comment:
This will take care of basic password security for you! The best thing is that this comes installed with Flask. Checkout the updates to the `User` model for how I've implemented this.